lib/addons/karpenter/index.ts

```diff
@@ -88,10 +98,10 @@ const RELEASE = 'blueprints-addon-karpenter';
 const defaultProps: HelmAddOnProps = {
     name: KARPENTER,
     namespace: KARPENTER,
-    version: '0.16.3',
+    version: 'v0.22.0',
     chart: KARPENTER,
-    release: RELEASE,
-    repository: 'https://charts.karpenter.sh',
+    release: KARPENTER,
```

Comment:
this release label change broke compatibility with existing helm charts, and `cdk deploy` now fails with error
> Received response status [FAILED] from custom resource. Message returned: Error: b'Release "karpenter" does not exist. Installing it now.\nError: rendered manifests contain a resource that already exists. Unabl
e to continue with install: PodDisruptionBudget "karpenter" in namespace "karpenter" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.he
lm.sh/release-name" must equal "karpenter": current value is "blueprints-addon-karpenter"\n'